utils/compression/compression.ts

```diff
@@ -0,0 +1,14 @@
+import LZString from "lz-string";
```

Comment:
Now that we know that this approach is viable, have you also considered using ZIP compression (i.e. what template-archive already does)?

Reply:
Yeah, We chose LZString compression for its simplicity and quick integration. Haven't properly explored the ZIP compression or using the template-archive approach fr. Perhaps might be something could revisit in next iterations, possibly in v2.